src/radical/entk/appman/appmanager.py

```diff
@@ -129,7 +136,8 @@ def resource_manager(self):
         :getter: Returns the resource manager object being used
         :setter: Assigns a resource manager
         """
-
+    
+        pprint.pprint(self._resource_manager)
         return self._resource_manager
```

Comment:
This is probably a debug print and should be removed, or converted to a log statement?